data/brands/amenity/restaurant.json

```diff
@@ -5773,9 +5773,9 @@
       "locationSet": {"include": ["jp"]},
       "tags": {
         "amenity": "restaurant",
-        "brand": "ステーキガスト",
```

Comment:
![图片](https://user-images.githubusercontent.com/42690037/218302989-6233f13c-f8c0-4c4c-9291-f5e422c0bbcd.png)

There are still a lot of name like this in current OSM data, if simple change brand name record, `ステーキガスト` will be collected to NSI in the future.

---
FYI: history of this entry in NSI: https://github.com/osmlab/name-suggestion-index/blame/02f82aa6da4f08369c85416b0eff7b73de92fa52/data/brands/amenity/restaurant.json#L5727-L5742

Reply:
This is because it is another variant of [ガスト](https://github.com/osmlab/name-suggestion-index/blob/02f82aa6da4f08369c85416b0eff7b73de92fa52/data/brands/amenity/restaurant.json#L5505), which is the main brand, and also because I referenced [JA:Naming sample](https://wiki.openstreetmap.org/wiki/JA:Naming_sample#amenity=restaurant).